### ISSUE TITLE
slim succeeds, but image does not run

### ISSUE THREAD

**Issue by icsy7867** [slimtoolkit/slim#475]
Expected Behavior
=================
Container runs

---
Actual Behavior
=================
Container does not run

---

Steps to Reproduce the Problem
===============================
 
This probably has to do with the container i made, but I am not sure where to start for debugging.

---
Specifications
=================
  - Version: 1.40
  - Platform: running in gitlab, using docker in docker.

So a bit of background...

I have a web app build from:
php:8.1-apache-bullseye

I add a few extra packages, and I compile the php sqlsrv driver from Microsoft.  Everything runs fine.

After a co-worker mentioned the app, I gave it a shot.

docker-slim build --http-probe=false --sensor-ipc-mode proxy password-share:8.1-bullseye
(Also tried it with --http-probe=true and the --sensor-ipc-mode because it's running in DIND via gitlab.)

Succeeded! Beautiful, tiny, image!

However, when I try to test and deploy onto my server, the image immediately crashes.  The logs in the container are simply:
`mkdir: cannot create directory '/var/lock': File exists`

If I change the --http-probe=true, I got a different error:
`/bin/sh: error while loading shared libraries: libc.so.6: cannot open shared object file: No such file or directory`

im 100% sure this is to do with my container and my lack of understanding with your tool.  But i am hoping someone can give me some guidance on steps to problem solve :D

Thank you!

**Comment by e7paul** [comment]
same problem

**Comment by waaman** [comment]
Hello, same thing also

**Comment by kcq** [comment]
@waaman are you also trying to minify a php-based app container image?

**Comment by kcq** [comment]
`mkdir: cannot create directory '/var/lock': File exists` - setting `--include-new` to `false` might have with this potentially, but it'll be good to learn more about the application. Great that we already have the base image info (`php:8.1-apache-bullseye`). Is this a web server app? What kind of endpoints does it have? If it has endpoints do you have an EXPOSE instruction in your Dockerfile? What port do you use? What does your app do?

**Comment by waaman** [comment]
> @waaman are you also trying to minify a php-based app container image?

Yes i try to create a slim version of a personal image based on **php:8-apache**.
II don't have enough skills on docker-slim, this is my first try.

>  Is this a web server app?
Yes it is.

> What kind of endpoints does it have?
The base image endpoint: apache2-foreground from **php:8-apache**

> do you have an EXPOSE instruction in your Dockerfile?
Yes i have 
EXPOSE 80
--http-probe=false is madatory if i want a slim version for it and the ended message if this error.

> What does your app do?
Just serve a php website

**Comment by waaman** [comment]
![slim lock exists](https://github.com/slimtoolkit/slim/assets/107315299/bf8b5d36-4370-47d7-9c5b-5fd14b637bcb)

**Comment by kcq** [comment]
try the latest version, 1.40.11, it has a flag to exclude the /var/lock data (`--exclude-varlock-files`) and the flag is enabled by default.

**Comment by tbreuss** [comment]
I have the same issue, `--exclude-varlock-files` doesn't help.

@waaman Were you able to solve the issue?
